### ISSUE TITLE
The only color is white for levels 1-14. Level 0 has color

### ISSUE THREAD

**Issue by Tower2k** [Parzivail-Modding-Team/LightLevel#8]
The only color is white for levels 1-14. Level 0 has color, red or yellow.

MC 1.18.1
lightlevel-1.17-1.18-3.0.0.jar
fabric-api-0.46.1+1.18.jar
![LightLevel](https://user-images.githubusercontent.com/86510859/152465721-e5ee1e1a-0a8d-4e2a-8c9a-d9d88f071d25.jpg)

**Comment by parzivail** [comment]
The lighting system, and by consequence the mob spawning system, was changed in 1.18 -- the colors should be accurate to where mobs can spawn. Can you verify that this is the case, or do mobs spawn in the white areas?

**Comment by Tower2k** [comment]
It's like that everywhere.  I have yet to see any 1-7 with red or yellow. 
Could it be that mobs don't spawn within 24 blocks of the player? That wouldn't make coloring useful at all.

**Comment by velnias75** [comment]
I use a `paper`-server and there it is possible to revert to the old lighting mob spawning behaviour.
An configuration option would be very nice.

**Comment by Graycot** [comment]
I can also confirm this issue:

![image](https://user-images.githubusercontent.com/72952069/159197915-90079250-63cb-4ed7-b3b3-7528d556d7ba.png)

Playing on Fabric 1.18.1

**Comment by parzivail** [comment]
@velnias75 Next update I'll be adding a color configuration option, which should take care of that

@Tower2k @Graycot Unless I have misunderstood, the mob spawning system changed in 1.18+ and the color reflects that. No mob can spawn on a block with light level 1+ under any condition (hence, white). On blocks with block light of zero and adequate skylighting, mobs can only spawn at night (yellow). With no skylighting and no block lighting, mobs can always spawn (red). This should reflect the pattern you see ingame. As I understand it, mobs can no longer spawn in light levels of 1 through 7.

**Comment by Graycot** [comment]
@parzivail Ah, my bad.  I did not realize that the mob spawning system changed in 1.18+. The mobs I witnessed in the lit up area must have spawned on light level 0 blocks and walked into the illuminated areas. 

I used an alt account and spectator mode to watch the spawning behavior of the mobs on my world. I did not observe any mobs spawning in the 1-7 light level blocks for the 10 minutes I watched.
